src/index.js

```diff
@@ -0,0 +1,208 @@
+        checkbox.addEventListener('change', () => {
+          changeElementToCompleted(index);
+        });
+      }
+    } else {
+      checkbox.addEventListener('change', () => {
+        changeElementToCompleted(index);
+      });
+    }
+  });
+}
+
+function addEventsToEditIcons() {
+  const editIcons = document.querySelectorAll('.edit-icon');
+  const todoElements = document.querySelectorAll('.todo-element');
+
+  todos.forEach((todo, index) => {
+    editIcons[index].addEventListener('click', () => {
+      const div = document.createElement('div');
+      div.classList.add('flex', 'todo-element');
+      div.style.backgroundColor = '#FFFBAE';
+
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.classList.add('checkbox');
+      checkbox.checked = todo.completed;
+
+      const input = document.createElement('input');
+      input.type = 'text';
+      input.classList.add('edit-input');
+      input.value = todo.description;
+      input.style.backgroundColor = 'transparent';
+
+      const span = document.createElement('span');
+      span.classList.add('material-icons', 'edit-icon');
+      span.style.marginLeft = 'auto';
+      span.style.cursor = 'pointer';
+      span.innerHTML = 'delete';
+
+      div.appendChild(checkbox);
+      div.appendChild(input);
+      div.appendChild(span);
+
+      todoElements[index].replaceWith(div);
+
+      input.addEventListener('keyup', (e) => {
+        if (e.key === 'Enter') {
+          const todo = todos[index];
+          todo.description = input.value;
+          updateTodo(todo, todos[index]);
+          const html = createReplaceTodoElement(todo);
+          div.innerHTML = html;
+          addEventsToEditIcons();
+          saveTodosLocally();
+          div.style.backgroundColor = 'white';
+        }
+      });
+
+      span.addEventListener('click', () => {
+        saveTodosLocally();
+        deleteTodo(todo, todos);
+        div.parentElement.remove();
+        saveTodosLocally();
+      });
+    });
+  });
+}
+
+window.addEventListener('load', () => {
+  const oldTodos = JSON.parse(localStorage.getItem('todos'));
+  if (oldTodos) {
+    todos = oldTodos;
+  }
+  populate();
+  addEventsToCheckboxes();
+  addEventsToEditIcons();
+});
+
+function addEventListenerToInput() {
+  const input = document.querySelector('#input');
+  input.addEventListener('keyup', (e) => {
+    if (e.key === 'Enter') {
+      const todo = new Todo(input.value, false, todos.length + 1);
+      addTodo(todo, todos);
+      addTodoElement(todo);
+      saveTodosLocally();
+      input.value = '';
+      addEventsToEditIcons(todos.length);
+      addEventsToCheckboxes(todos.length - 1);
+    }
+  });
+}
+
+addEventListenerToInput();
+
+button.addEventListener('click', () => {
+  const todoElements = document.querySelectorAll('.todo-element');
+  const removedTodos = [];
+  for (let i = 0; i < todos.length; i += 1) {
+    if (todos[i].completed === true) {
```

Comment:
What do you think about using a conditional shorthand, it is gonna be shorter and achieve the same result.
If(todo[i].completed) {
// ...do something
}